packages/sdk/src/oracle/types/interfaces.ts

```diff
@@ -0,0 +1,25 @@
+import { BigNumberish, BigNumber, Signer, TransactionResponse, TransactionReceipt } from "../types/ethers";
```

Comment:
this file defines the interfaces that are needed to generalize the oracle service